features.json

```diff
@@ -0,0 +1,8 @@
+{
+    "routerTransform": {
+      "SALESFORCE": true,
+      "GA": true,
+      "MARKETO": true,
+      "HUBSPOT":true
```

Comment:
Lets use the name here. For Hubspot the name is `HS`. Also, let's format the file.

Reply:
made relevant changes